packages/teams-js/src/internal/handlers.ts

```diff
@@ -157,7 +162,7 @@ export function registerHandlerHelper(
  */
 export function registerOnThemeChangeHandler(handler: (theme: string) => void): void {
   HandlersPrivate.themeChangeHandler = handler;
-  handler && sendMessageToParent('registerHandler', ['themeChange']);
+  handler && sendMessageToParentWithVersion('v2', 'registerHandler', ['themeChange']);
```

Comment:
Similar question to one I left in communication: why are these v2? Shouldn't they be v1 because they're the first version we're tracking telemetry of?

Reply:
@TrevorJoelHarris please take a look on comment that I left for 'initialize' function.

This place keeps the same logic. register* functions are public APIs in Teams-Js library V1.X.Y. Thus, those in Teams-Js library V1.X.Y will be labeled as V1. Then, in Teams-Js library V2.X.Y, those register related APIs are moved to each capability, i.e. registerFullScreenHandler API is moved to Pages capability, so, for now, it needs to be labeled as V2.